serialization.md

```diff
@@ -0,0 +1,48 @@
+# CloudEvents Serialization Profile - Version 0.1
+
+This document specifies how a CloudEvent is to be serialized into certain
+encoding formats. Compliant CloudEvents implementations that support these
+formats MUST adhere to these rules for these formats.
+
+## Table of Contents
+- [Notational Conventions](#notational-conventions)
+- [JSON](#json)
+
+## Notational Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to
+be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## JSON
+
+When serialized in [JSON](https://tools.ietf.org/html/rfc7159) a CloudEvent
+MUST adhere to the following format:
+
+```
+{
+  "cloud-events-version": "cloud-events-version value",
```

Comment:
I'm curious whether it would be more useful to use valid fake values rather than prose in quotes.

Reply:
I struggled with this because if we put fake values in there then we need something to say "property X from the spec goes into json property 'x'".  While in many cases it should be really really obvious, I felt that a spec should be more explicit about it.  If all we cared about was JSON I actually would have proposed that we have the serialization of each property next to its definition in the spec itself. But I didn't want to assume everyone would be ok with JSON getting special treatment.

Reply:
@inlined I added an example to try to address your concern. See if that helps.